packages/ra-ui-materialui/src/list/filter/FilterForm.tsx

```diff
@@ -30,35 +24,45 @@ import { FilterFormInput } from './FilterFormInput';
 import { FilterContext } from '../FilterContext';
 
 export const FilterForm = (props: FilterFormProps) => {
-    const { defaultValues, filters: filtersProps, ...rest } = props;
```

Comment:
`defaultValues` is not used in all RA's code, and I tested it and it does not work, so I removed it to reduce tech debt. Shouldn't be a BC since it does not currently work.